src/ActionBmwCodeCell.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2023 Inclusive Design Research Centre, OCAD University
+ * All rights reserved.
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/inclusive-design/adaptive-palette/blob/main/LICENSE
+ */
+
+import { html } from "htm/preact";
+import { BlissSymbol, BciAvId } from "./BlissSymbol";
+import "./ActionBmwCodeCell.scss";
+
+type ActionBmwCodeCellProps = {
+  id: string,
+  options: {
+    label: string,
+    columnStart: number,
+    columnSpan: number,
+    rowStart: number,
+    rowSpan: number,
+    bciAvId: BciAvId
+  }
+}
+
+export function ActionBmwCodeCell (props: ActionBmwCodeCellProps) {
+
+  const { columnStart, columnSpan, rowStart, rowSpan } = props.options;
+  const { bciAvId, label } = props.options;
```

Comment:
These two lines can be combined into one line as they are all parsing `props.options`.

Reply:
The reason they were split is that putting them all on one line is too long.  I've found a way to split a single destructuring statement across the two lines.